CHANGES_NEXT_RELEASE

```diff
@@ -7,6 +7,5 @@
 - Add: new value for URI param 'options': unique - like 'values', but avoiding duplicates (Issue #1050)
 - Add: check min length for IDs in v2 (Issue #1836)
 - Fix: PATCH /v2/entities correct interpretaion of null value and missing type in payload
-. Fix: PATCH /v2/entities missing value field (Issue #1789)
-
-
+- Fix: PATCH /v2/entities missing value field (Issue #1789)
+- Fix: parsing 'q-string-filter' in early stages and storing resulting objects for mongo-backend to access (q-part of Issue #1705)
```

Comment:
Correct, the user should not care about these changes, seeing the broker as a 'black box'.
I'll remove the line from CNR.